README.md

```diff
@@ -515,4 +520,61 @@ type FooType = {};
 type foo = {};
 ```
 
+## Rules (suppressing)
+
+These are rules that suppress problems.
+
+
+### `define-flow-type`
```

Comment:
It looks like you have added these examples by hand? I might be wrong. Just in case, there is a command to add assertion examples to the documentation, [`npm run readme-add-assertions`](https://github.com/gajus/eslint-plugin-flowtype/blob/733e0761025a6835e9732703bb026af816a72649/package.json#L31). This command is executed as part of `npm run readme`.